docs/dev-proc/smbios-rules.md

```diff
@@ -0,0 +1,64 @@
+# Dasharo Product Guidelines: SMBIOS Information
+
+## Introduction
+
+This document describes the guidelines for filling in the SMBIOS fields in
+devices supported by Dasharo. It is meant to be used primarily as a reference
+for Dasharo developers.
+
+## BIOS Information (Type 0)
+
+### Vendor field
+
+The vendor field is always set as follows:
+
+```
+3mdeb Embedded Systems Consulting
```

Comment:
```suggestion
3mdeb
```

This is correct name of the company. Maybe we should mentioned, that for firmware before 13 January 2022 this is as above?

Reply:
Fixed